README.md

```diff
@@ -8,10 +8,20 @@
 docker build -t micdenny/rabbitmq-windows:3.6.9 ./rabbitmq/windows/3.6.9
 docker build -t micdenny/rabbitmq-windows:3.6.12 ./rabbitmq/windows/3.6.12
 docker build -t micdenny/rabbitmq-windows:3.8.16 ./rabbitmq/windows/3.8.16
-docker build -t micdenny/rabbitmq-windows:3.9.8 ./rabbitmq/windows/3.9.8
+docker build -t micdenny/rabbitmq-windows:3.9.8 ./rabbitmq/windows/3.9.
+
 docker build -t micdenny/rabbitmq-windows:latest ./rabbitmq/windows/latest
 ```
 
+#### Build for specific Windows Server versions 
+```powershell
+docker build -t micdenny/rabbitmq-windows:3.8.35-servercore-ltsc2019 ./rabbitmq/windows/3.8.35 --build-arg SERVER_VERSION=ltsc2019 
+docker build -t micdenny/rabbitmq-windows:3.8.35-servercore-ltsc2012 ./rabbitmq/windows/3.9.35 --build-arg SERVER_VERSION=ltsc2022 
+docker build -t micdenny/rabbitmq-windows:3.9.22-servercore-ltsc2019 ./rabbitmq/windows/3.9.22 --build-arg SERVER_VERSION=ltsc2019 
+docker build -t micdenny/rabbitmq-windows:3.9.22-servercore-ltsc2012 ./rabbitmq/windows/3.9.22 --build-arg SERVER_VERSION=ltsc2022 
+docker build -t micdenny/rabbitmq-windows:3.10.7-servercore-ltsc2019 ./rabbitmq/windows/3.10.7 --build-arg SERVER_VERSION=ltsc2019 
+docker build -t micdenny/rabbitmq-windows:3.10.7-servercore-ltsc2012 ./rabbitmq/windows/3.10.7 --build-arg SERVER_VERSION=ltsc2022
+```
 ### Redis
```

Comment:
at this point I would write only a couple of sample, instead of trying adding all the possible combination, so something like this, but maybe written better as you think could be better and concise:

## RabbitMQ
```powershell
docker build -t micdenny/rabbitmq-windows:x.y.z ./rabbitmq/windows/x.y.z
docker build -t micdenny/rabbitmq-windows:x.y.z ./rabbitmq/windows/latest
```

### Build for specific Windows Server versions
```powershell
docker build -t micdenny/rabbitmq-windows:x.y.z-servercore-ltsc2019 ./rabbitmq/windows/x.y.z --build-arg SERVER_VERSION=ltsc2019 
docker build -t micdenny/rabbitmq-windows:x.y.z-servercore-ltsc2012 ./rabbitmq/windows/x.y.z --build-arg SERVER_VERSION=ltsc2022 
```

## Redis
```powershell
docker build -t micdenny/redis-windows:x.y ./redis/windows/x.y
```